Userland/Utilities/disasm.cpp

```diff
@@ -6,8 +6,11 @@
 
```

Comment:
Can we put the disassembly dumps in the PR desc or a comment only? I'm not sure how much value they add being in the commit description itself.